### PR TITLE
BOAC-3709: admit page now says 'no data' where a data element is blank or null

### PR DESCRIPTION
**Pull request by cesarvh**

### CODE REVIEW

**Comment by cesarvh** [comment]
JIRA: https://jira.ets.berkeley.edu/jira/browse/BOAC-3709

**Comment by cesarvh** [comment]
image
Added dashes to represent the missing data similar to the non-admit student cohort vue. I didn't add it to the CEP column because there would be way too much going on, though.

File: src/components/admit/AdmitStudentRow.vue [hunk 1]
```diff
@@ -26,7 +26,15 @@
     <td>
       <div>
         <span class="sr-only">C E P</span>
-        <span :id="`row-${rowIndex}-special-program-cep`">{{ admitStudent.specialProgramCep }}</span>
+        <span
+          v-if="admitStudent.specialProgramCep === '' || $_.isNil(admitStudent.specialProgramCep)"
+          :id="`row-${rowIndex}-special-program-cep`"
+        ><span class="sr-only">No data</span></span>
+
+        <span
```
Comment (johncrossman) [inline src/components/admit/AdmitStudentRow.vue:34]: @cesarvh - Looks good. Did you intend to have em-dash (for sighted users) above. You have it below. Note: we often use `&mdash;`

Comment (cesarvh) [reply src/components/admit/AdmitStudentRow.vue:34]: Fixed!

**Comment by johncrossman** [comment]
Thank you @cesarvh   Waiting on Travis.

**Pull request merged**